core/vm/interpreter.go

```diff
@@ -184,7 +184,7 @@ func (in *EVMInterpreter) Run(contract *Contract, input []byte, readOnly bool) (
 		}
 		// Get the operation from the jump table and validate the stack to ensure there are
 		// enough stack items available to perform the operation.
-		op = contract.GetOp(pc)
+		op = contract.GetOp(contract.CodeBeginOffset() + pc)
```

Comment:
Could also use that `contract.OpCodeAt(udest)` I suggested above

Reply:
Could also just change `GetOp` to the new semantics, couldn't it? It seems to be the only place it is being used at (and has internal length checks).